plugin/trino-hive/src/main/java/io/trino/plugin/hive/s3/S3SecurityMapping.java

```diff
@@ -58,7 +59,8 @@ public S3SecurityMapping(
             @JsonProperty("accessKey") Optional<String> accessKey,
             @JsonProperty("secretKey") Optional<String> secretKey,
             @JsonProperty("useClusterDefault") Optional<Boolean> useClusterDefault,
-            @JsonProperty("endpoint") Optional<String> endpoint)
+            @JsonProperty("endpoint") Optional<String> endpoint,
+            @JsonProperty("roleSessionName") Optional<String> roleSessionName)
```

Comment:
please move next to `iamRole`. Disallow to set this property when `iamRole` is empty.